frontend/helpers/next-helpers.ts

```diff
@@ -2,7 +2,6 @@ import { PROD_USER_AGENT } from '@config/constants';
 import { CACHE_DISABLED } from '@config/env';
 import { setSentryPageContext } from '@ifixit/sentry';
 import { withTiming } from '@ifixit/helpers';
-import { clearCache } from '@lib/cache';
```

Comment:
It looks like this was the only usage of this `clearCache` function, so we could drop the definition too.